packages/devextreme-cli/templates/react/application/src/themes/metadata.additional.json

```diff
@@ -5,5 +5,8 @@
   "outputColorScheme": "additional",
   "makeSwatch": true,
   "base": true,
-  "widgets": ["treeview", "navbar"]
+  "widgets": [
```

Comment:
Maybe it should be changed in angular and vue templates accordingly?

Reply:
Done